common/spec/dummy_package_manager/version.rb

```diff
@@ -4,6 +4,21 @@
 
 module DummyPackageManager
   class Version < Gem::Version
+    def initialize(version)
```

Comment:
Why is this the same as `Dependabot::GithubActions::Version` now?

In the tests for `local_tag_for_pinned_version`, I'm using a `git-upload-pack` captured from `actions/cache` https://github.com/actions/checkout/tags

Having the active `version_class` (which in that test - is `DummyPackageManager::Version`) understand those tags was the path of least resistance.
Alternatively I could setup and capture a cleaner fixture - I liked the idea of the test case being "real".